executor/point_get.go

```diff
@@ -237,6 +237,14 @@ func (e *PointGetExecutor) Next(ctx context.Context, req *chunk.Chunk) error {
 				return err
 			}
 
+			readFromCache := !e.ctx.GetSessionVars().IsPessimisticReadConsistency()
```

Comment:
I don't really get the name `readFromCache`... In the RC case, we cannot read from cache?

To my understanding, it is somewhat (whether to) `lockNonExistIdxKey`? (Correct me if I get it wrong)